packages/mediacenter/kodi/package.mk

```diff
@@ -175,6 +175,8 @@ configure_package() {
 
   if target_has_feature neon; then
     KODI_NEON="-DENABLE_NEON=ON"
+  elif [ "${TARGET_ARCH}" = "x86_64" ]; then
```

Comment:
If we go this route, it's better to invert condition and check if it's arm or arm64. Other architectures don't have NEON anyway.

Reply:
👍 inverted the logic. Checking for aarch64 and arm first.